no/Mortgage/v0/schema/schema.yaml

```diff
@@ -0,0 +1,784 @@
+'$schema': 'http://json-schema.org/draft-07/schema#'
+'$id': 'http://open-broker.org/no/Mortgage/v0/schema'
+
+definitions:
+  reference:
+    '$id': '#reference'
+    title: reference
+    description: |
+      A reference used for identification of an event. Consisting of two
+      parts, an issuer domain name of the form `com.example`, and an
+      arbitrary id string. The values, taken together must be unique.
+      In other words, the issuing organization can issue an ID once only.
+    type: object
+    additionalProperties: false
+    properties:
+      issuer:
+        type: string
+        title: Domain-name of the issuer of the reference, in reverse order
+        description: Reversed DNS name for the broker, for example, example.com becomes com.example
+        pattern: "^[a-zA-Z0-9]+\\.(([a-zA-Z0-9_-])|([a-zA-Z0-9_-]\\.[a-zA-Z0-9_-]))+$"
+      id:
+        type: string
+        title: "Reference number for the broker"
+        description: "An arbitrary reference string that is unique"
+  MortgageApplicationCreated:
+    title: Application created
+    description: |
+      An event indicating the creation of an application for
+      a private unsecured loan.
```

Comment:
```suggestion
      a mortgage.
```

Reply:
Nice catch, thank you! (really nice feature with commit suggestions, yay!)

Reply:
(i also love commit suggestion! srsly life easier <3)